actions/class.Tests.php

```diff
@@ -251,6 +252,7 @@ public function authoring()
                     );
                 }
                 if ($this->getRequestParameter('originResourceUri') !== null) {
+                    $this->getTranslationSyncService()->syncById($this->getRequestParameter('originResourceUri'));
```

Comment:
@shaveko the FE already calls and endpoint `translation/sync` to call the `syncByRequest`. I am not sure also asking the BE to do the same is a good idea 🤔

Reply:
please check [comment](https://oat-sa.atlassian.net/browse/ADF-1827?focusedCommentId=288492) to the ticket

Reply:
Now yes, if we remove the FE part of the code, it makes sense to do it on the BE side. When I checked the PRs, there was only the BE part introduced, so the call would be redundant and this sync process is quite expensive.